src/Altinn.Platform/Altinn.Platform.Events/Events/Repository/PostgresRepository.cs

```diff
@@ -227,5 +229,89 @@ public async Task<Subscription> GetSubscription(int id)
                 await _conn.CloseAsync();
             }
         }
+
+        /// <inheritdoc/>
+        public async Task<List<Subscription>> GetSubscriptionsExcludeOrg(string source, string subject, string type)
+        {
+            List<Subscription> searchResult = new List<Subscription>();
+            try
+            {
+                await _conn.OpenAsync();
+
+                NpgsqlCommand pgcom = new NpgsqlCommand(getSubscriptionsExcludeOrgsSql, _conn);
+                pgcom.Parameters.AddWithValue("source", NpgsqlDbType.Varchar, source);
+                pgcom.Parameters.AddWithValue("subject", NpgsqlDbType.Varchar, subject);
+                pgcom.Parameters.AddWithValue("type", NpgsqlDbType.Varchar, type);
+
+                using (NpgsqlDataReader reader = pgcom.ExecuteReader())
+                {
+                    while (reader.Read())
+                    {
+                        Subscription subscription = new Subscription();
+                        subscription.Id = Convert.ToInt32(reader["id"].ToString());
+                        subscription.SourceFilter = new Uri(reader["sourcefilter"].ToString());
+                        subscription.SubjectFilter = reader["subjectfilter"].ToString();
+                        subscription.TypeFilter = reader["typefilter"].ToString();
+                        subscription.Consumer = reader["consumer"].ToString();
+                        subscription.EndPoint = new Uri(reader["endpointurl"].ToString());
+                        subscription.CreatedBy = reader["createdby"].ToString();
+                        subscription.Created = DateTime.Parse(reader["time"].ToString());
+                        searchResult.Add(subscription);
```

Comment:
One tip from SonarCloud is that these lines could have been a method. They are the same lines as in other methods.